modules/fluid_properties/include/userobjects/SodiumSaturationFluidProperties.h

```diff
@@ -0,0 +1,200 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "SinglePhaseFluidProperties.h"
+
+/**
+ * Fluid properties for liquid sodium at saturation conditions \cite{sas} \cite{fink}.
+ */
+class SodiumSaturationFluidProperties : public SinglePhaseFluidProperties
+{
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Woverloaded-virtual"
+public:
+  static InputParameters validParams();
+
+  SodiumSaturationFluidProperties(const InputParameters & parameters);
+
+  /**
+   * Fluid name
+   *
+   * @return "sodium"
+   */
+  virtual std::string fluidName() const override;
+
+  /**
+   * Density from pressure and temperature
+   *
+   * @param[in] p   pressure (Pa)
+   * @param[in] T   temperature (K)
+   * @return density (kg/m$^3$)
+   */
```

Comment:
You don't have to repeat the doxygen in the inherited classes. We have a flag that will use the doco from the parent if it is missing in the child. I know we talked about this before and it is nice to open a header file and see what is the meaning of all the params - especially when somebody new opens the file. I assume people would go to the parent class for this or the online doco.

This is up to you if you want to change it. I am ok either way. Just saying this is not really needed.

Reply:
yeah you're right, I just copied another class in here that I made a while ago before realizing that - I'll remove all the extra doxygen